src/views/views.less

```diff
@@ -108,6 +108,12 @@
   button {
     margin-right: 4px;
   }
+  .blank-state-pf-helpbutton {
+    border-left: 0;
+    border-right: 0;
```

Comment:
Adding `btn` above lets you remove the border setting.
However, you'll need to set `vertical-align: baseline`

Reply:
fixed